hangupsbot/plugins/image_screenshot/__init__.py

```diff
@@ -88,6 +87,18 @@ def screenshot(bot, event, *args):
         yield from bot.coro_send_message(event.conv_id, "<i>processing another request, try again shortly</i>")
         return
 
+    if _internal['browser'] is None:
```

Comment:
This tri-state requires the bot or plugin to be re-loaded. What if the error was transient?